api/src/main/java/org/openmrs/module/pihcore/setup/LocationTagSetup.java

```diff
@@ -239,6 +240,7 @@ private static void setStandardHaitiSingleSiteLocationTags(LocationService locat
         setLocationTagsFor(locationService, LocationTags.REGISTRATION_LOCATION, Arrays.asList(location));
         setLocationTagsFor(locationService, LocationTags.VITALS_LOCATION, Arrays.asList(location));
         setLocationTagsFor(locationService, LocationTags.MENTAL_HEALTH_LOCATION, Arrays.asList(location));
+        setLocationTagsFor(locationService, LocationTags.MCH_LOCATION, Arrays.asList(location));
```

Comment:
Thanks @BranfordTGbieor this block is for Haiti, but it is strange that we weren't tagging non-Mirebalais sites in Haiti as MCH locations, so I agree we likely want to add this, correct @lnball ?

Reply:
MCH is only at Mirebalais (and usually for J9).  Hopefully this will change in the future.  FYI @mogoodrich

Reply:
Cool, thanks @lnball ... however, thinking about it, we can go ahead and set up the tag here (that is, keep this change) I think?  As long as we don't have the MCH component enabled for a site, the link still won't appear?

Reply:
That's good @mogoodrich .  We can add the location for all the Haiti (ZL) sites, but use the component to toggle the features.

Reply:
Oops, will adjust ASAP! An oversight it was!

Reply:
I think it's actually fine to leave this in @BranfordTGbieor  because it prepares us for when we do roll out MCH elsewhere in Haiti.